Add post for personal-allowance (#259)
UBICenter/ubicenter.org: _posts/2021-07-25-personal-allowance.md

---
layout: post
current: post
cover: assets/images/personal-allowance.png
navigation: True

> title: Funding UBI by eliminating the personal allowance

Reviewer: ```suggestion
title: Funding UBI by eliminating the UK personal allowance
```

date: 2021-06-21

> tags: [uk,ubi]

Reviewer: it's all `ubi` here ;)
```suggestion
tags: [uk]
```

subclass: 'post'
author: ines

> excerpt: A budget-neutral plan exchanging tax exemption for universal basic income

Reviewer: ```suggestion
excerpt: The budget-neutral reform would lower poverty by up to 29%.
```

class: post-template
---

> <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>

Reviewer: remove these 3 lines and instead add `useplotly: true` in the front matter

<script src="https://requirejs.org/docs/release/2.3.5/minified/require.js"></script>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>

> # Replacing the UK personal allowance with a universal basic income

Reviewer: remove these two lines

## This budget-neutral proposal could decrease poverty by up to 29%.
Among several potential pay-fors in their recent [UBI discussion paper](https://d3n8a8pro7vhmx.cloudfront.net/libdems/pages/1811/attachments/original/1621669347/145_-_Universal_Basic_Income.docx_%281%29.pdf?1621669347), the Liberal Democrats considered lowering the personal allowance from its current value of £12,570 to £2,500. That is, the first £2,500 of an individual's annual earnings would be exempted from taxation, rather than the first £12,570. [As my colleague Nikhil Woodruff found](https://www.ubicenter.org/lib-dem-policy-paper), this would raise about £60 billion which, when funding a UBI, significantly lowers poverty and inequality.

> The Liberal Democrats were not the first to connect the personal allowance and UBI. The Green Party has also proposed effectively exchanging the two policies[^1], and several think tanks have modeled UBIs involving reductions or repeals of the personal allowance. While the personal allowance is highly salient— a 2014 poll found that [85% of the public](https://www.ipsos.com/ipsos-mori/en-uk/personal-allowances-rise-most-popular-conference-season-tax-pledges) supported raising it—it's also a large source of potential revenue for the costly proposition of paying every member of society every month.

Reviewer: footnote outside punctuation (please apply to other footnotes throughout)

[^1]: The Green Party states that in their model, UBI will be taxable, but ["all income tax payers will have a tax-free allowance which is the equivalent to their Universal Basic Income amount"](https://www.greenparty.ie/wp-content/uploads/2018/07/Green-Party-Universal-Basic-Income-Policy.pdf); this means that, in practice, the UBI would not be taxed and the personal allowance would effectively be eliminated.  

### How the personal allowance works

The personal allowance was [introduced in 1979](http://taxhistory.co.uk/Income%20Tax%20Allowances.htm), and it has grown almost every year since. Liberal Democrats accelerated its growth by [calling](https://www.standard.co.uk/news/politics/libdems-to-let-1-3m-low-earners-avoid-paying-tax-8506838.html) for expanding it from £10,000 in 2015 to £12,500 in 2020; this succeeded, and it now sits at £12,570, where it is [expected to remain until April 2026](https://www.reuters.com/article/uk-health-coronavirus-britain-budget-inc-idUSKBN2AV1LP).

> While many countries have a tax-free band like the personal allowance, the UK also makes the more uncommon decision to phase it out for higher earners. Individuals earning above £100,000 lose £1 of personal allowance for every £2 in earnings, phasing out fully for those earning £125,140 per year and creating 62% marginal tax rates for earners in this range.

Reviewer: more uncommon -> uncommon

    id3 = id2/income_with_PA.groupby(income.decile_rank()).mean()[n] * 100
    decile_change.append(id3.round(2))

change = pd.Series(decile_change)
chart1 = format_fig((px.bar(y=decile_change, labels={"x":"Decile", "y":"Income gain"}).update_layout(
    title_text='Income increase brought by the personal allowance for households in each income decile',
    xaxis_title ="Household income decile",
    yaxis_title ="Change in household net income",
    showlegend= False,
    yaxis_ticksuffix="%",
    xaxis = dict(
        tickvals = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9],
        ticktext = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
    ),
    
).update_traces(marker_color='#1976D2')))
chart1
    </code>
  </pre>
</div>

<script>
function show_code_8() {
  var x = document.getElementById("code_block_8");
  if (x.style.display === "none") {
    x.style.display = "block";
  } else {
    x.style.display = "none";
  }
}
</script>

<div>
  <script>
    $(document).ready(function(){
      $("#graph_graph_8_1").load("{{site.baseurl}}assets/markdown_assets/personal-allowance/graph_8_1.html");
    });
  </script>
</div>
<div id = "graph_graph_8_1"></div>

<div>
  <script>
    $(document).ready(function(){

>       $("#graph_graph_8_2").load("{{site.baseurl}}assets/markdown_assets/personal-allowance/graph_8_2.html");

Reviewer: title is too long, gets cut off

how about `Effect of personal allowance on household net income by decile`?

>“[Universal credit recipients] will not see the full tax gain of £130 from the increase in the personal allowance; instead, they will only gain overall by £48.10, as their Universal Credit award will be reduced by £81.90. However, those earning above £11,850 who receive tax credits will benefit from the full £130 because tax credits are based on gross income.”

Similarly, the [Resolution Foundation](https://www.resolutionfoundation.org/app/uploads/2014/12/Missing-the-target1.pdf) found that the Liberal Democrats’ plan to increase the personal allowance from £10,600 in 2015 to £12,500 by 2020 gave only around £18 of additional annual income to the bottom 10% of households and £203 to the top 10%.
### Replacing the personal allowance with UBI

While the personal allowance is only slightly progressive by itself, using revenue from repealing the personal allowance to fund a UBI is highly progressive. Fully repealing the Personal Allowance could fund a weekly UBI of £29 per person, and this lowers the Gini index from 0.386 to 0.369.

Each £2,000 decrease in the personal allowance could fund an increase in the UBI amount of around £110 annually, or just over £2 a week.

> <button onclick="show_code_11()">Click to show code</button>

Reviewer: reverse x axis for consistency

    BI_amount = (revenue_diff/(sim.calc("people").sum()))/52
    
    return BI_amount.round()

UBI_amounts = [find_ubi_amount(i) for i in (PA_amounts)]

PA_text = ["12500", "11500", "10500", "9500", "8500", "7500", "6500", "5500", "4500", "3500", "2500", "1500", "500", "0"]
chart = format_fig(px.line(x=PA_text, y=UBI_amounts, labels={"x": "Personal allowance amount", "y":"UBI amount"}).update_layout(
    title_text='Amount of UBI at different levels of personal allowance reduction',
    xaxis_title ="Amount of personal allowance (£)",
    yaxis_title ="UBI (£)",
).update_yaxes(range=[-0.1, 30]).add_hline(y=0, line_dash="dash", line_color="#BDBDBD").add_vline(x=0, line_dash="dash", line_color="#BDBDBD"))

    </code>
  </pre>
</div>

<script>
function show_code_11() {
  var x = document.getElementById("code_block_11");
  if (x.style.display === "none") {
    x.style.display = "block";
  } else {
    x.style.display = "none";
  }
}
</script>

<div>
  <script>
    $(document).ready(function(){
      $("#graph_graph_11_1").load("{{site.baseurl}}assets/markdown_assets/personal-allowance/graph_11_1.html");
    });
  </script>
</div>
<div id = "graph_graph_11_1"></div>

> This policy would, in general, serve as an income transfer from the richest 40% to the poorest 60%. You can adjust the personal allowance amount with the slider to see how partial personal allowance replacements fare: with further cuts to the personal allowance and higher universal basic incomes, reforms achieve stronger redistributive effects.

Reviewer: ensure all decile values are visible on x axis

df_list = []
sim_num = 0
age = sim.calc("age", map_to ="person")
for i in PA_amounts2:
    df = pd.DataFrame({"Age": ages, "Income change": income_by_age(sim_list[sim_num]), "PA amount": i})
    df_list.append(df)
    sim_num += 1
    
final_df = pd.concat(df_list)
final_df
format_fig(px.bar(final_df, x="Age", y="Income change", animation_frame="PA amount", range_y=[-1100,3100],
                 hover_data={"PA amount": False}).update_layout(
    title_text='Average change in household net income by age',
    yaxis_title="Change in household net income",
    yaxis_tickprefix="£",
    xaxis_title="Age",
    showlegend= False,
    ).update_traces(marker_color='#1976D2'))
    </code>
  </pre>
</div>

<script>
function show_code_31() {
  var x = document.getElementById("code_block_31");
  if (x.style.display === "none") {
    x.style.display = "block";
  } else {
    x.style.display = "none";
  }
}
</script>

<div>
  <script>
    $(document).ready(function(){
      $("#graph_graph_31_1").load("{{site.baseurl}}assets/markdown_assets/personal-allowance/graph_31_1.html");
    });
  </script>
</div>
<div id = "graph_graph_31_1"></div>

#### Impacts for specific household types

> Overall, 52% of people come out ahead, most of them belonging to the bottom four deciles. 91% of the lowest decile, 73% of the second-lowest decile, and 62% of the third-lowest decile would see their incomes rise by over 5%. In contrast, most people who see their income decrease are in the upper deciles and lose less than 5%.

Reviewer: just a note that using the slider on this chart creates an awesome visual effect, love it

> Overall, 52% of people come out ahead, most of them belonging to the bottom four deciles. 91% of the lowest decile, 73% of the second-lowest decile, and 62% of the third-lowest decile would see their incomes rise by over 5%. In contrast, most people who see their income decrease are in the upper deciles and lose less than 5%.

Reviewer: Clarifying that this refers to full repeal:
```suggestion
Entirely replacing the personal allowance with UBI would benefit 52% on a net basis, most of them belonging to the bottom four deciles. 91% of the lowest decile, 73% of the second-lowest decile, and 62% of the third-lowest decile would see their incomes rise by over 5%. In contrast, most people who see their income decrease are in the upper deciles, and these people mostly lose less than 5%.
```

            l.append(tmp)
    return pd.concat(l).reset_index()

intra = intra_decile_graph_data(baseline, *sims)

GREY = "#BDBDBD"

COLORS = ("#616161", GREY, "#F5F5F5", "#C5E1A5", "#558B2F",)[::-1]

intra_graph = format_fig(
    px.bar(
        intra,
        x="fraction",
        y="decile",
        orientation="h",
        color="Outcome",
        animation_frame="PA",
        color_discrete_sequence=COLORS,
    ).update_layout(
        yaxis_tickvals=list(range(1, 11)),
        xaxis_tickformat="%",
        yaxis_title="Decile of household equivalized net income",
        xaxis_title="Share of people",
        title="How replacing the personal allowance with a UBI affects people within deciles",
    )
)
intra_graph
    </code>
  </pre>
</div>

<script>
function show_code_35() {
  var x = document.getElementById("code_block_35");
  if (x.style.display === "none") {
    x.style.display = "block";
  } else {
    x.style.display = "none";
  }
}
</script>

<div>
  <script>
    $(document).ready(function(){
      $("#graph_graph_35_1").load("{{site.baseurl}}assets/markdown_assets/personal-allowance/graph_35_1.html");
    });
  </script>
</div>
<div id = "graph_graph_35_1"></div>

> A single individual living alone would see their income rise when they make under £17,000 and over £117,000.

Reviewer: Hide series other than baseline and reform net income

> A single individual living alone would see their income rise when they make under £17,000 and over £117,000.

Reviewer: Slider says reform= instead of PA=

> A single individual living alone would see their income rise when they make under £17,000 and over £117,000.

Reviewer: Remove `value` from y axis title, just leave blank by setting to `""`

<button onclick="show_code_47()">Click to show code</button>
<div id="code_block_47" style="display: none;">
  <pre>
    <code>
#Calc
mtr_two_kids = graphs.mtr_chart(reform_list, names=reform_names, variables=["household_net_income", "tax", "benefits"],
                situation_function=couple_two_children)
    </code>
  </pre>
</div>

<script>
function show_code_47() {
  var x = document.getElementById("code_block_47");
  if (x.style.display === "none") {
    x.style.display = "block";
  } else {
    x.style.display = "none";
  }
}
</script>

<button onclick="show_code_48()">Click to show code</button>
<div id="code_block_48" style="display: none;">
  <pre>
    <code>
#Graph
format_fig(mtr_two_kids.update_layout(
    title_text='Marginal tax rate for a single person when replacing the personal allowance with a UBI'))
    </code>
  </pre>
</div>

<script>
function show_code_48() {
  var x = document.getElementById("code_block_48");
  if (x.style.display === "none") {
    x.style.display = "block";
  } else {
    x.style.display = "none";
  }
}
</script>

### Conclusion

> My research aligns with other analyses showing the progressivity of replacing the personal allowance with UBI. In 2019, the New Economics Foundation [modeled](https://neweconomics.org/2019/03/nothing-personal) replacing the personal allowance with a budget-neutral Weekly National Allowance paid to adults and an increase in the child benefit, making the payments non-taxable but included in means tests. They found progressive results, stating it would “[lift] 200,000 families out of poverty and shift the £8bn currently spent on tax allowances for the 35% highest income families to the remaining 65% of families”. Compass also [modeled](https://www.compassonline.org.uk/wp-content/uploads/2019/03/Compass_BasicIncomeForAll_2019.pdf) replacing the personal allowance with a budget-neutral flat payment of £25 a week for children and adults of working age, as well as a flat rate citizen’s pension of £164.35. They too found progressive results, with the Gini coefficient decreasing from 0.337 to 0.365. Most recently, [my colleagues found](https://www.ubicenter.org/progressive-adjustments-lib-dem-working-group) that the policy in the Liberal Democrats discussion paper could be made more progressive by repealing all, rather than part, of the personal allowance.

Reviewer: move period inside quotes

> Beyond these other papers, I've found that the impacts occur roughly linearly on the margin, meaning that the policy concept can operate incrementally. Furthermore, the policy disproportionately benefits children, and separate studies show that this will in turn improve educational and health outcomes. The impacts are largely but not uniformly beneficial to low income households, though losses are small.

Reviewer: link "separate studies" to child-allowance.ubicenter.org/empirical

> Given these findings, UBI proponents in the UK should take a serious look at whether keeping the personal allowance is worthwhile, provided the benefits of the UBI that could be funded through eliminating it.

Reviewer: Feel free to adjust, but in general looking to reduce normativity/prescriptiveness (also considering that our audience extends beyond UBI proponents)
```suggestion
Reforms that broaden the tax base are often regressive on their own, but how the revenue is spent matters. Repealing the personal allowance is one such instance: using the revenue to fund a UBI switches the distributional sign from regressive to highly progressive.
```
